modules/apps/object/object-admin-rest-impl/rest-openapi.yaml

```diff
@@ -872,6 +874,71 @@ paths:
                             schema:
                                 $ref: "#/components/schemas/ObjectRelationship"
             tags: ["ObjectRelationship"]
+    "/object-definitions/by-external-reference-code/{objectDefinitionExternalReferenceCode}/object-validation-rules":
```

Comment:
As we just discussed, please change the path parameter name from `objectDefinitionExternalReferenceCode` to `externalReferenceCode`